src/StackAdmin/Azs.Fabric.Admin/readme.md

```diff
@@ -51,44 +51,128 @@ require:
-  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ScaleUnitNode.json
-  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/SlbMuxInstance.json
-  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/StorageOperationResults.json
-  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2018-10-01/Drive.json
   - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2018-10-01/StorageSubSystem.json
-  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2018-10-01/Volume.json
+  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2019-05-01/Drive.json
+  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2019-05-01/Volume.json
+```
 
 ### File Renames 
+``` yaml
 module-name: Azs.Fabric.Admin 
 csproj: Azs.Fabric.Admin.csproj 
 psd1: Azs.Fabric.Admin.psd1 
 psm1: Azs.Fabric.Admin.psm1
-
-subject-prefix: ''
-module-version: 0.0.1
+```
 
 ### Parameter default values
+``` yaml
 directive:
   - where:
       parameter-name: ResourceGroupName
     set:
       default:
         script: -join("System.",(Get-AzLocation)[0].Name)
+
+  # Rename Get-AzsFileShare to Get-AzsInfrastructureShare
+  - where:
+      subject: FileShare
+    set:
+      subject: InfrastructureShare
+
+  # Rename model property names for StorageSubSystem to match spec
+  - where:
+      model-name: StorageSubSystem
+      property-name: TotalCapacityGb
+    set:
+      property-name: TotalCapacityGB
+
+  - where:
+      model-name: StorageSubSystem
+      property-name: RemainingCapacityGb
+    set:
+      property-name: RemainingCapacityGB
+  
+  # Rename model property names for Drive to match spec
+  - where:
+      model-name: Drive
+      property-name: CapacityGb
+    set:
+      property-name: CapacityGB
+
+  # Rename model property names for Volume to match spec
+  - where:
+      model-name: Volume
+      property-name: TotalCapacityGb
+    set:
+      property-name: TotalCapacityGB
+
+  - where:
+      model-name: Volume
+      property-name: RemainingCapacityGb
+    set:
+      property-name: RemainingCapacityGB
+
+  - where:
+      model-name: Volume
+      property-name: Label
+    set:
+      property-name: VolumeLabel
+
+  # Rename cmdlet parameter name in InfrastructureShare
+  - where:
+      subject: InfrastructureShare
+      parameter-name: FileShare
+    set:
+      parameter-name: Name
+
+  # Rename cmdlet parameter name in StorageSubSystem
+  - where:
+      subject: StorageSubSystem
+      parameter-name: StorageSubSystem
+    set:
+      parameter-name: Name
+
+  # Rename cmdlet parameter name in Drive
+  - where:
+      subject: Drive
+      parameter-name: Drive
+    set:
+      parameter-name: Name
```

Comment:
I believe all these renaming you are doing matching to the existing module? @zyxyoshine

Reply:
yes